packages/integration/lib/src/bloc/post/handler/new_post_event_handler.dart

```diff
@@ -78,6 +78,24 @@ void newPostEventHandler(
           }
 
           if (media.attachmentType == LMMediaType.video) {
+            String? thumbnailURL;
+            String? thumbnailPath = await LMFeedVideoThumbnail.thumbnailFile(
+              video: mediaFile.path,
+              quality: 8,
```

Comment:
what does the `quality: 8` mean, compress the video quality to 8/100 , or anything else?

Reply:
8/10, 80% of the original size

Reply:
got it.